<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Numerico</class>
 <widget class="QMainWindow" name="Numerico">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>807</width>
    <height>725</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #D8FFFF, stop: 1 #ECFFFF);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="objLabelEnterNum">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string>                         Enter a number:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="objInputUpperRangeLimit">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #E0E0E0, stop: 1 #FFFFFF);
    border: 1px solid gray;
    border-radius: 5px;
    margin-top: 0.5ex;  
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="objBtnConfirm">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>0</y>
      <width>124</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    background-color: red;
	border-width: 2px;
    border: 2px solid gray;
    border-radius: 9px;
    
    border-style: outset;
    border-color: beige;
    font: bold 14px;
    min-width: 6em;
}
margin-top: 1ex; 
                                               </string>
    </property>
    <property name="text">
     <string>Confirm</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="objCurrentCard">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>321</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit{
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #E0E0E0, stop: 1 #FFFFFF);
    border: 2px solid gray;
    border-radius: 5px;
    margin-top: 1ex;  
}</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="objHistory">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>320</y>
      <width>731</width>
      <height>281</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit{
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #E0E0E0, stop: 1 #FFFFFF);
    border: 2px solid gray;
    border-radius: 5px;
    margin-top: 1ex;  
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="objLabelCardsNumber">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>391</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>220</y>
      <width>321</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox#groupBox{
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #E0E0E0, stop: 1 #FFFFFF);
    border: 2px solid gray;
    border-radius: 5px;
    margin-top: 1ex;  
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top center; /* position at the top center */
    padding: 0 3px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF0ECE, stop: 1 #FFFFFF);
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QRadioButton" name="objRadioFound">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>20</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 10);
border-radius: 5px;</string>
     </property>
     <property name="text">
      <string>Yes</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="objRadioNotFound">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>50</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 10);
border-radius: 5px;</string>
     </property>
     <property name="text">
      <string>No</string>
     </property>
    </widget>
    <widget class="QLabel" name="objLabelQuestion">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>201</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 10);
border-radius: 5px;</string>
     </property>
     <property name="text">
      <string>    Does current card contains 
     your imagine number?</string>
     </property>
    </widget>
    <zorder>objLabelQuestion</zorder>
    <zorder>objRadioFound</zorder>
    <zorder>objRadioNotFound</zorder>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>60</y>
      <width>341</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox#groupBox_2{
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #E0E0E0, stop: 1 #FFFFFF);
    border: 1.3px solid gray;
    border-radius: 8px;
    margin-top: 2ex;  
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top center; /* position at the top center */
    padding: 0 3px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF0ECE, stop: 1 #FFFFFF);
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLineEdit" name="objInputNumberInCard">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>341</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
  background-color: rgb(224,224,224);
   border: 2px solid gray;
    border-radius: 7px;
    
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="objPlainTextEditFound">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>321</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPlainTextEdit{
   background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #D8FFFF, stop: 1 #ECFFFF);
    border: 1.3px solid gray;
    border-radius: 3px;
    margin-top: 0.5ex;  
}</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="objPlainTextEditNotFound">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>321</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPlainTextEdit{
   background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #D8FFFF, stop: 1 #ECFFFF);
    border: 1.3px solid gray;
    border-radius: 3px;
    margin-top: 0.5ex;
	margin-bottom:0.5ex;  
}</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
